src/plugins/mer/merconstants.h

```diff
@@ -29,11 +29,19 @@
 namespace Mer {
 namespace Constants {
 
+#ifdef MER_BASE_OS_NAME
+const char MER_OS_NAME[] = QT_TRANSLATE_NOOP("Mer", MER_BASE_OS_NAME " OS");
+const char MER_SDK_NAME[] = QT_TRANSLATE_NOOP("Mer", MER_BASE_OS_NAME " SDK");
+#else
+const char MER_OS_NAME[] = "Sailfish OS";
+const char MER_SDK_NAME[] = "Sailfish SDK";
+#endif
+
```

Comment:
This seems wrong - one branch is i18n-enabled, the other is not. Is is necessary to enable i18n for these at all?

This also makes me think, do we really need to add a custom mechanism to change the branding? Wouldn't it be more convenient to use i18n for that. I mean you can leave Sailfish OS/Sailfish SDK in the code and provide translations (easy to automate that) that say something else.

Reply:
Of course this comment should be posted under the new PR...